lte/gateway/c/oai/tasks/sgw/sgw_task.c

```diff
@@ -260,6 +262,7 @@ static void sgw_exit(void)
 {
   OAILOG_DEBUG(LOG_SPGW_APP, "Cleaning SGW\n");
 
+  gtp_tunnel_ops->uninit();
```

Comment:
We can call gtpv1u_exit here instead of exposing the gtp_tunnel_ops variable here. Then we don't need to include the header gtpv1u.h here.